Jenkinsfile

```diff
@@ -1,69 +1,64 @@
-if (env.BRANCH_NAME == 'master') {
-    properties([
-        pipelineTriggers([
-            upstream(
-                upstreamProjects: 'utils/master',
-                threshold: hudson.model.Result.SUCCESS,
-            ),
-            // Build fresh base images every day at 9 PM.
-            // The time of day doesn't really matter on this, but when it builds
-            // successfully it triggers other builds which could have failures,
-            // so this should probably be during a time when people are awake.
-            cron('0 21 * * *'),
-        ]),
-    ])
-}
-
+pipeline {
+  // TODO: Make this cleaner: https://issues.jenkins-ci.org/browse/JENKINS-42643
+  triggers {
+    upstream(
+      upstreamProjects: (env.BRANCH_NAME == 'master' ? 'utils/master' : ''),
+      threshold: hudson.model.Result.SUCCESS,
+    )
 
-try {
-    // check out code
-    stage name: 'check-out-code'
-
-    node('slave') {
-        dir('src') {
-            checkout scm
-        }
-        stash 'src'
-    }
+    // Build fresh base images every day at 9 PM.
+		// The time of day doesn't really matter on this, but when it builds
```

Comment:
fix formatting?

Reply:
Thanks, looks like there were some rogue tabs in there that I missed, thanks for catching those!